website/docs/docs/core/pip-install.md

```diff
@@ -36,10 +38,31 @@ alias env_dbt='source <PATH_TO_VIRTUAL_ENV_CONFIG>/bin/activate'
 ```
 
 ### Installing the adapter
-Once you know [which adapter](/docs/supported-data-platforms) you're using, you can install it as `dbt-<adapter>`. For example, if using Postgres:
+
+Once you decide [which adapter](/docs/supported-data-platforms) you're using, you can install using the command line. Beginning in v1.8, you must install both dbt Core and the desired adapter because dbt-core has been decoupled from the adapter installation.
+
+<VersionBlock firstVersion="1.8">
+
+```shell
+python3 -m pip install dbt-core dbt-ADAPTER_NAME. 
```

Comment:
@dbeatty10 or @colin-rogers-dbt I changed this to say `python3` is this right? I couldn't get `python` to work but perhaps this is my local set up? I noticed the requirements state Python 3+.

Reply:
It's reasonable to either leave it as `python` or update it to `python3`. Either way, the instructions are bound to not work for someone though 😢

If we update it to `python3`, we should update all the same places as in https://github.com/dbt-labs/docs.getdbt.com/pull/4498.

There's some more background on the differences between these names here:
- https://www.reddit.com/r/learnpython/comments/mf7t0n/why_python3_in_command_prompt_vs_python/
- https://stackoverflow.com/questions/64801225/python-or-python3-what-is-the-difference

Python 2 reached end of life on January 1st, 2020, and so it's been reasonable to use just `python` rather than `python3` since then.

Reply:
+1 we should use `python`, while mac defaults to python2 (perhaps what's happened on your computer) python3 is the default so no need to specify

Reply:
OK happy to change it! Thanks for your input.

Reply:
Changed in https://github.com/dbt-labs/docs.getdbt.com/pull/5002/commits/bbe77e00a1a6c9d395fe3c531612f0ccabd9772f